src/elements/styles.js

```diff
@@ -358,6 +358,64 @@ class ApplyStyles {
       this.setStyle('placeholderFocus', 'display', 'none');
     }
   }
+
+  applyBackgroundImageStyles(target) {
+    const targetStyles = [
+      'background_image_url',
+      'background_image_display',
+      'background_image_layout',
+      'background_image_vertical_layout',
+      'background_image_size',
+      'background_image_size_x',
+      'background_image_size_y',
+      'background_image_repeat'
+    ];
+
+    this.apply(target, targetStyles, (...styles) => {
```

Comment:
these styles have to be broken down into individual apply statements based on style groups. if you look at what apply does under the hood, it doesn't set mobile breakpoint styles if no mobile override exists for any of the styles passed in. but if at least one of the styles do exist, the mobile breakpoint CSS will be applied entirely.

passing all of these styles into the same apply function can create an unnecessary number of mobile overrides